cg/meta/upload/scout/raredisease_config_builder.py

```diff
@@ -61,11 +61,13 @@ def build_load_config(self) -> RarediseaseLoadConfig:
         self.include_case_files(load_config)
         self.get_sample_information(load_config)
         self.include_pedigree_picture(load_config)
-        load_config.custom_images = self.load_custom_image_sample()
+        custom_images = self.load_custom_image_sample()
+        if custom_images:
+            load_config.custom_images = custom_images
         load_config.human_genome_build = GenomeBuild.hg19
         return load_config
 
-    def load_custom_image_sample(self) -> CustomImages:
```

Comment:
Will this ever return None? Seems as if the only possibility for that happening would be if there were no samples for the case?